src/features/task-list/ui.tsx

```diff
@@ -0,0 +1,14 @@
+import { Task } from "@entities/task/model"
```

Comment:
Actually, I think it's not a feature. It's just `ui` slice of task entity. I would don't create a particular component for it. I would create a `TaskItem` component into the `entities/task`. Although you can create an `ui.tsx` file in the `entities/task` and implement two components there. In the one file you can create a `list` component and `item` component too. It's not usual practice, but I did it. And personally for me, it's a good solution. Because of the `list` and `items` are really bind. So, I suggest to move the list to `entities` and create an `Item` stub component there too